modules/react/button/lib/parts/ButtonContainer.tsx

```diff
@@ -142,6 +142,8 @@ export const ButtonContainer = styled('button', {
         };
       case 'extraSmall':
         return {
+          ...type.levels.subtext.medium,
+          fontWeight: type.properties.fontWeights.bold,
```

Comment:
This change will affect all Primary, Secondary, and Tertiary buttons, but it something we missed when we added the `type.levels.subtext.medium` in previous PRs. The default font-weight for that size is 400, so we need bump it up to 700.